operator/src/test/java/oracle/kubernetes/operator/DomainProcessorTest.java

```diff
@@ -972,6 +972,22 @@ void whenClusterScaleDownToReplicas1_fromReplicas2_server1And2And3WithAlwaysPoli
     assertThat(info.getClusterService(CLUSTER), notNullValue());
   }
 
+  @Test
+  void whenClusterScaleDownToReplicas1_fromReplicas2_reportIncompleteEvent()
```

Comment:
suggest a better name might be:
```
whenScalingDown_reportIncompleteEvent
```

It is actually relevant to the functionality that we are going from 2 to 1, rather than some other change?

Reply:
I followed the naming patterns in the other similar test cases. Yes, I can change it to the suggested name.